pyat/at/lattice/elements.py

```diff
@@ -972,6 +972,70 @@ def __init__(self, family_name: str, m66=None, **kwargs):
         super(M66, self).__init__(family_name, M66=m66, **kwargs)
 
 
+class SimpleQuantDiff(_DictLongtMotion, Element):
+    """
+    Linear tracking element for a simplified quantum diffusion,
+    radiation damping and energy loss
+    """
+    _BUILD_ATTRIBUTES = Element._BUILD_ATTRIBUTES
+    default_pass = {False: 'IdentityPass', True: 'SimpleQuantDiffPass'}
+
+    def __init__(self, family_name: str, beta_x: Optional[float]=1.0,
+                 beta_y: Optional[float]=1.0, emit_x: Optional[float]=0.0,
+                 emit_y: Optional[float]=0.0, sigma_dp: Optional[float]=0.0,
+                 tau_x: Optional[float]=0.0, tau_y: Optional[float]=0.0,
+                 tau_z: Optional[float]=0.0, U0: Optional[float]=0.0,
+                 **kwargs):
+        """
+        Args:
+            family_name:    Name of the element
+            
+        Optional Args:
```

Comment:
You should put the defaults values there and add that emit=0, etc..... will be ingnored (similar to the sentence you have for the ring function)
In fact it would be nice to have all docstings identical for the same input